mitm.go

```diff
@@ -178,6 +183,10 @@ func (p *Proxy) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	p.proxyMITM(sc, cc)
 }
 
+func SkipNone(req *http.Request) bool {
```

Comment:
Also basic doc here? (Or you can make it private; it's just nice if all public functions have documentation.)